services/web/server/src/simcore_service_webserver/catalog/_models.py

```diff
@@ -54,43 +53,29 @@ def get_html_formatted_unit(
 #
 # Transforms from catalog api models -> webserver api models
 #
-
-
-# Caching:  https://cachetools.readthedocs.io/en/latest/index.html#cachetools.TTLCache
-# - the least recently used items will be discarded first to make space when necessary.
+# Uses aiocache (async) instead of cachetools (sync) in order to handle concurrency better
+# SEE https://github.com/ITISFoundation/osparc-simcore/pull/6169
 #
+_SECOND = 1  # in seconds
+_MINUTE = 60 * _SECOND
+_CACHE_TTL: Final = 1 * _MINUTE
 
-_CACHE_MAXSIZE: Final = int(
-    os.getenv("CACHETOOLS_CACHE_MAXSIZE", "100")
-)  # number of items  i.e. ServiceInputGet/ServiceOutputGet instances
-_CACHE_TTL: Final = int(os.getenv("CACHETOOLS_CACHE_TTL_SECS", "60"))  # secs
-
-
-def _hash_inputs(
-    service: dict[str, Any],
-    input_key: str,
-    *args,  # noqa: ARG001 # pylint: disable=unused-argument
-    **kwargs,  # noqa: ARG001 # pylint: disable=unused-argument
-):
-    return f"{service['key']}/{service['version']}/{input_key}"
-
-
-def _cachetools_cached(*args, **kwargs):
-    def decorator(func):
-        if os.getenv("CACHETOOLS_DISABLE", "0") == "0":
-            return cachetools.cached(*args, **kwargs)(func)
-        _logger.warning("cachetools disabled")
-        return func
 
-    return decorator
+def _hash_inputs(_f, *_args, **kw):
+    assert _f.__name__  # nosec
+    assert not _args  # nosec
```

Comment:
```suggestion
    assert not _args, "the callables has positional arguments, this is not supported"  # nosec
```

Reply:
I think `assert not _args` is quite readable. My assumption at this point is that the function does not have positional arguments